### ISSUE TITLE
Doesn't work?

### ISSUE THREAD

**Issue by trusktr** [atom-minimap/minimap-find-and-replace#25]
Doesn't seem to highlight any results. Maybe it is outdated and needs to be updated for the latest find/replace?

**Comment by Rion1337** [comment]
I've found If you hit control f, escape, then control f again it will highlight your search results the second time but it does not live update as you alter your search.

**Comment by BitGrub** [comment]
BeefaloKing's suggestion works as an impractical workaround, hope someone figures out what the issue is

**Comment by Barni-Pack** [comment]
Any solution for this one?

**Comment by gmatocha** [comment]
I have the same problem, and BeefaloKing's workaround works. I think this might have something to do with the operation of Find since I also use jfrottet's find-list plugin and it has the same problem and the workaround works for it too.
I'm using Atom 1.49.0, if that helps.

**Comment by second-pair** [comment]
I have the same issue and @BeefaloKing 's workaround works.  I tried installing Atom 1.52.0 x64 in a fresh Windows Sandbox, installed just "minimap" and "minimap-find-and-replace" plugins and the issue persisted.  I then installed "find-list", which exhibits similar behaviour, explicitly stating it can't find any results until closing search and re-opening it again.  So I think @gmatocha might be on to something.
